syntax = "proto3";

import "numproto/protobuf/ndarray.proto";

package hellonumproto;

service NumProtoServer {
  rpc SayHelloNumProto(NumProtoRequest) returns (NumProtoReply) {}
}

message NumProtoRequest {
  numproto.protobuf.NDArray arr = 1;
}

message NumProtoReply {
  numproto.protobuf.NDArray arr = 1;
}
